src/ScriptBuilder/Meters/MeterDefinition.cs

```diff
@@ -10,12 +10,12 @@ public MeterDefinition(string name, string description, string unit, string[] ta
             Tags = tags;
         }
 
-        public string Description { get; }
+        public readonly string Description;
 
-        public string Name { get; }
+        public readonly string Name;
 
-        public string Unit { get; }
+        public readonly string Unit;
```

Comment:
@danielmarbach as at the moment `Unit` and `Tags` are not used we get inspection error suggesting that we should make those fields private. We could: 
 - suppress the error (which I don't like) 
 - mark the fields as private
 - remove the fields from this class as we are not using them (I am leaning towards this soltuion)